runtime/mainnet/src/lib.rs

```diff
@@ -626,6 +711,8 @@ mod runtime {
 	pub type Balances = pallet_balances::Pallet<Runtime>;
 	#[runtime::pallet_index(11)]
 	pub type TransactionPayment = pallet_transaction_payment::Pallet<Runtime>;
+	#[runtime::pallet_index(12)]
+	pub type Treasury = pallet_treasury::Pallet<Runtime>;
```

Comment:
Might want to move under Governance, only as that is where the treasury UI is under on PJS.

Reply:
will address in a subsequent PR that will refactor this into a proper config location